Source/Core/Core/DolphinAnalytics.h

```diff
@@ -61,6 +61,15 @@ enum class GameQuirk
 
   USES_WD_UNIMPLEMENTED_IOCTL,
 
+  // Some games use invalid/unknown graphics commands (see e.g. bug 10931).
+  // These are different from unknown opcodes: it is known that BP/CP/XF is being used to,
```

Comment:
is being used to what?

Reply:
er, I guess that should be "being written to", or perhaps just "being used".